stripe/api_resources/sigma/scheduled_query_run.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import, division, print_function
+
+from stripe.api_resources.abstract import UpdateableAPIResource
+from stripe.api_resources.abstract import ListableAPIResource
+
+
+class ScheduledQueryRun(ListableAPIResource):
+    OBJECT_NAME = 'scheduled_query_run'
```

Comment:
The object name should really be `sigma.scheduled_query_run` based on recent API decisions we made. It's not though, similarly to apple pay domains.

The issue is that if we change this in the future, we'll need to have some custom logic for the deserializer which will be tricky. I don't think we have renamed an object before.

This comment is mostly here as food for thoughts though!